consensus/polybft/commitment.go

```diff
@@ -4,33 +4,24 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
-	"math"
 	"math/big"
 	"reflect"
 
 	"github.com/0xPolygon/polygon-edge/crypto"
 	"github.com/0xPolygon/polygon-edge/state/runtime/precompiled"
 	"github.com/0xPolygon/polygon-edge/types"
 	"github.com/mitchellh/mapstructure"
-	"github.com/umbracle/ethgo"
 	"github.com/umbracle/ethgo/abi"
 )
 
```

Comment:
WDYT about keeping this file named as `state_transaction.go`? Rationale for it is because it not only contains logic for ABI encode/decode of commitments but also uptime transactions.

Reply:
Doing it in https://github.com/0xPolygon/polygon-edge/pull/1013/commits/1fc911037c5f7204aeb4daadac360cfb69f9fcd9 because it's of the scope of this PR but in the line of separating responsibilities in the code base, this deserves a refactor IMO splitting this to files, and even with the contractsapi gen.